src/ui/tabpopup.c

```diff
@@ -840,7 +843,8 @@ meta_select_workspace_get_type (void)
 }
 
 static void
-meta_select_workspace_class_init (MetaSelectWorkspaceClass *klass)
+meta_select_workspace_class_init (MetaSelectWorkspaceClass *klass,
```

Comment:
Does this really produce warning? I don't get warnings for class_init building metacity with gcc 8.2.

Reply:
I'm sorry, I used gcc 9... I will remove the version in the PR title.
```
$ cat /etc/fedora-release 
Fedora release 30 (Rawhide)
$ gcc --version
gcc (GCC) 9.0.1 20190209 (Red Hat 9.0.1-0.4)
```

Reply:
```
ui/tabpopup.c:827:9: error: cast between incompatible function types from ‘void (*)(MetaSelectWorkspaceClass *)’ {aka ‘void (*)(struct _MetaSelectWorkspaceClass *)’} to ‘void (*)(void *, void *)’ [-Werror=cast-function-type]
  827 |         (GClassInitFunc) meta_select_workspace_class_init,
```

Reply:
I can't see the direct/indirect call to `meta_select_workspace_class_init` in metacity:
```
[robert@localhost metacity]$ git grep meta_select_workspace_class_init
src/ui/select-workspace.c:meta_select_workspace_class_init (MetaSelectWorkspaceClass *class)
```

Reply:
Ah, that is win for me... :D I have "modernized" that part... :)